public/cron.ctrlPlanning.php

```diff
@@ -35,12 +35,14 @@
 
 // chdir($path) : important pour l'execution via le cron
 chdir($path);
+require_once "$path/vendor/autoload.php";
 
-require_once "$path/include/config.php";
-require_once "$path/include/function.php";
-require_once "$path/absences/class.absences.php";
-require_once "$path/planning/postes_cfg/class.tableaux.php";
-require_once "$path/postes/class.postes.php";
+use App\PlanningBiblio\Framework;
+
+require_once "$path/public/include/config.php";
+require_once "$path/public/include/function.php";
+require_once "$path/public/absences/class.absences.php";
+require_once "$path/public/postes/class.postes.php";
 
```

Comment:
This file should go to /src/Cron/Legacy/
and $path should be remplaced by \__DIR__ . '/../../../public'